### ISSUE TITLE
Thumbnail generation leaves many defunct chrome processes

### ISSUE THREAD

**Issue by simonbaird** [simonbaird/tiddlyhost#297]
The `dumb-init` might be relevant, see [here](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker).

Could also perhaps try to move the entire thumbnail generation into its own container.

**Comment by simonbaird** [comment]
As a workaround I'm restarting the job processor container several times a day.

**Comment by jandresrodriguez** [comment]
@simonbaird you found any solution for the defunct chrome processes ?

**Comment by simonbaird** [comment]
@jandresrodriguez Nope. I'm still restarting containers every few hours to clear them.

**Comment by simonbaird** [comment]
Not sure if this is still happening.
